eng/helix/content/InstallYarn.cmd

```diff
@@ -0,0 +1,7 @@
+set PATH=%HELIX_WORKITEM_ROOT%;%PATH%;%HELIX_CORRELATION_PAYLOAD%\jdk\bin
+call npm.cmd i yarn
```

Comment:
Why is this script structured this way? In particular, it seems like you are installing yarn with npm then backing up the local installed dependencies then installing with yarn from the package.json.

Reply:
Yeah yarn and npm each nuke each other, so ideally we would only use one or the other, but that's not currently the case in this PR, so this prevents stomping